libp2p.go

```diff
@@ -52,7 +52,7 @@ func ChainOptions(opts ...Option) Option {
 // - If no peerstore is provided, the host is initialized with an empty
 // peerstore.
 //
-// Canceling the passed context will stop the returned libp2p node.
```

Comment:
Is this the only correct order to call these actions? What happens if we call Close before we cancel the context?

Reply:
@raulk The order is NOT important. I've updated the PR.